src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -275,4 +280,19 @@ private function addFlashMessageProductV2IsDisabled(): void
             )
         );
     }
+
+    private function addFlashMessageNotMultistoreCompatible(): void
+    {
+        $this->addFlash(
+            'error',
+            $this->trans(
+                'This page is not yet compatible with the multistore feature. To access the page, please %sdisable the multistore feature%s.',
```

Comment:
```suggestion
                'This page is not yet compatible with the multistore feature. To access the page, please [1]disable the multistore feature[/1].',
```

We usually use this kind of placeholders to inject HTML code

Reply:
I was not sure, I saw that `%s` was used for the same purpose in the same controller so I went for consistency, but ok 👍